Recorder/RecordDetailView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RecordDetailView.swift
+//  Recorder
+//
+//  Created by 김보승 on 2022/06/13.
+//
+
+import SwiftUI
+
+struct RecordDetailView: View {
+    
+//    let music: Music
+    
+    var body: some View {
+        ZStack {
+            Color.background.edgesIgnoringSafeArea(.all)
+            VStack {
+                HStack {
+                    Spacer()
+                    Image("backwindow")
+                }.padding(.top, 70.0)
+                Spacer()
+            } // BackgroundView용 VStack End
+            HStack {
+                VStack(alignment: .leading) {
+                    Text("일이삼사오육칠팔구십일이삼사오육") // "music.title"
+                        .font(.title2)
+                        .fontWeight(.bold)
+                        .foregroundColor(.titleBlack)
+                        .multilineTextAlignment(.leading)
+                        .padding(.bottom, 3)
+                    
+                    Text("일이삼사오육칠팔구십일이삼사오육칠팔구십일") // "music.artist"
+                        .font(.body)
+                        .fontWeight(.regular)
+                        .foregroundColor(.titleDarkgray)
+                    Spacer()
+                }// MusicInform VStack End
+                .padding(.leading, 5)
+                Spacer()
+            } // MusicImform HStack End
+            .padding(.leading, 23)
+            VStack {
+    
+                HStack(alignment: .center) {
+                    
+                    Button(action: {}, // action 내에 클릭시 모달을 통해 이미지 띄우는 기능 추가 예정
```

Comment:
```suggestion
                    Button(action: {}, // TODO: action 내에 클릭시 모달을 통해 이미지 띄우는 기능 추가 예정
```

// TODO: 는 Xcode안에서 앞으로 해야할일을 작성할 때 사용해요 : )

Reply:
넵 해당 방식으로 작성하겠습니다